clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -183,6 +239,22 @@ public DescribeTopicsResult describeTopics(Collection<String> topicNames) {
     /**
      * Describe some topics in the cluster.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link DescribeTopicsResult#values() values()} method of the  returned {@code DescribeTopicsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.UnknownTopicOrPartitionException}
+     *     if the authorized topic was in metadata first, but was not found during later look up</li>
```

Comment:
not used here (InvalidTopicException is used instead)

Reply:
The request is the same for listTopics, so it is the same here